src/amp/components/AdConsent.tsx

```diff
@@ -1,97 +1,122 @@
 import React from 'react';
-import { css, cx } from 'emotion';
-import { palette } from '@guardian/src-foundations';
-import { textSans } from '@guardian/src-foundations/typography';
-import Tick from '@frontend/static/icons/tick.svg';
+import { css } from 'emotion';
+import { palette, space } from '@guardian/src-foundations';
+import { textSans, headline } from '@guardian/src-foundations/typography';
+import { Button } from '@guardian/src-button';
+import { ThemeProvider } from 'emotion-theming';
+import { brand } from '@guardian/src-foundations/themes';
+import { SvgCheckmark } from '@guardian/src-icons';
 import { JsonScript } from './JsonScript';
 
 const consentUIStyle = css`
-    ${textSans.medium()};
-    color: ${palette.neutral[97]};
-    background-color: ${palette.neutral[20]};
+    ${textSans.small()};
+    color: ${palette.brandText.primary};
+    background-color: ${palette.brandBackground.primary};
     max-width: 600px;
+    padding: ${space[4]}px;
     margin: 0 auto;
     overflow-x: hidden;
-`;
 
-const containerDivStyle = css`
-    margin: 16px;
-`;
+    h2 {
+        color: ${palette.brandText.primary};
+        ${headline.small()};
+    }
 
-const h2Style = css`
-    margin-bottom: 16px;
-    font-size: 24px;
-    font-weight: bold;
-`;
+    a {
+        color: inherit;
+    }
 
-const pStyle = css`
-    margin-bottom: 8px;
-`;
+    .center {
```

Comment:
would this be better expressed as a new css block, rather than using the cascade cc @gtrufitt?

Reply:
I read somewhere that a single level down was acceptable?

Reply:
if it says it it means it 👍 :)